CONTRIBUTING.md

```diff
@@ -238,7 +238,11 @@ You can build and start the API reference website by running these two commands
 
 #### Docs website
 
-You can build and start a local docs website by running these two commands:
+You can build and start a local docs website by running:  
+
+`npm run docs-website-build-run`
```

Comment:
would be cool in the future (not this PR) to have an additional combo command to run the docs & the API docs at the same time

Reply:
Sorry, I meant something like (name is just an example):

```
"docs-run-docs-and-api": "npm run docs-runLocalDocker && npm run docs-api-build-run"
```

This command would allow you to run the documentation + the API docs so you can browse between both. Right now when editing locally, if you start from the docs and click on "API Docs" on the left navigation menu it gives you a 404 because there's nothing on the `/api` path.

But again, we can do it at some point in the future!